specifications/service-registry.yaml

```diff
@@ -0,0 +1,278 @@
+openapi: 3.0.0
+info:
+  description: 
+    API example for service registry. The API provides methods of distributing
+    and searching metadata related to mobility services. The API provides a way
+    for the service providers to publish their services and the the MaaS-operators
+    and end users to query available services and the user API standards in any 
+    area of interest. The services can be filtered by used API specifications 
+    and the certification proviced by trusted third parties. The API also provides
+    methods for distributing the registered metadata between multiple instances
+    of the service.
+    
+  version: "0.1.0-oas3"
+  title: Service registry
+  contact:
+    name: FLOU
+    url: 'https://www.flou.io/'
+    email: sami.makinen@flou.io
+  license:
+    name: CC BY 4.0
```

Comment:
This repository uses an Apache license, which is similar to CC-by for software. Can we change this to use the Apache license?